src/app/plugins/register.plugin.ts

```diff
@@ -68,6 +68,12 @@ export class RegisterPlugin extends Plugin {
     if (invalidClasses.length > 0) {
       messageForUser += `\nUnable to locate these classes: ${invalidClasses.join(' ')}`;
     }
-    message.reply(messageForUser);
+    this.container.messageService
+      .sendTextMessage(message, messageForUser, { reply: true, delimiter: ' ' })
+      .catch((err) =>
+        this.container.loggerService.warn(
+          `Failed to give feedback to ${message.author.username} abouts registering to classes. Error info:\n${err}`
+        )
+      );
```

Comment:
Not necessary, the user is only allowed to register for 5 classes at max, at a time.